version.go

```diff
@@ -0,0 +1,5 @@
+package mgob
```

Comment:
This should in `main` not `mgob`, as per `ldflags "-X main.version=$APP_VERSION"`. Also the version var is lowercase but you've changed it to uppercase.

Reply:
I've changed to use an exported `Version` constant in the `mgob` package. I think this might be the most suitable way to do it, but if its not please feel free to do a commit to better! :)